### PR TITLE
Update FS-1032-fsharp-in-dotnet-sdk.md

### PR DESCRIPTION
**Pull request by KevinRansom**
Add section describing the net sdk templates

### CODE REVIEW

**Comment by dsyme** [comment]
Thank you @KevinRansom this is fantastic detail

@enricosada If you're up for it can I ask you to review.  Also @cartermp , @nosami, @forki, @Krzysztof-Cieslak you may want to take a look

File: RFCs/FS-1032-fsharp-in-dotnet-sdk.md [hunk 1]
```diff
@@ -133,6 +133,133 @@ Relevant comment from @dsyme
...
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Program.fs" />
+  </ItemGroup>
+</Project>
+````
```
Comment (dsyme) [inline RFCs/FS-1032-fsharp-in-dotnet-sdk.md]: Could you show a few PackageReference?

Could you list any deviations from the C# format please?  

Are there any issues about resources and resource embedding, e.g. existing deviations of the F# project format, see https://github.com/Microsoft/visualfsharp/issues/1050?

Comment (KevinRansom) [inline RFCs/FS-1032-fsharp-in-dotnet-sdk.md]: There are no deviations, beyond source code globbing not working for obvious reasons, package refs are as per C# or VB in the SDK we do nothing different.  Beyond the extra properties to control FSharp version, FSharp package and the valuetuple stuff.  Everything else is vanilla.

Comment (dsyme) [inline RFCs/FS-1032-fsharp-in-dotnet-sdk.md]: Super - please include that information then, thanks.  It's important to call that out :)

File: RFCs/FS-1032-fsharp-in-dotnet-sdk.md [hunk 2]
```diff
@@ -133,6 +133,139 @@ Relevant comment from @dsyme
 > 
 > I was so excited by this result I told my wife.  That’s a sure indicator that we have to make sure we keep that value. 
 
+### DotNet SDK F# Templates
```
Comment (cartermp) [inline RFCs/FS-1032-fsharp-in-dotnet-sdk.md]: Nit: rename to ".NET SDK"

File: RFCs/FS-1032-fsharp-in-dotnet-sdk.md [hunk 3]
```diff
@@ -133,6 +133,139 @@ Relevant comment from @dsyme
...
+valid values include but are not limited to:
+* netcoreapp1.0
+* netstandard1.6
+* net45
+* net46
+* net461
+* net47
+
+__TargetFrameworks__
+The dotnetsdk allows a project to produce multiple targets use the target frameworks tag
```
Comment (cartermp) [inline RFCs/FS-1032-fsharp-in-dotnet-sdk.md]: "dotnetsdk" ---> ".NET SDK"

"target frameworks tag" ---> "`TargetFrameworks` tag"

**Comment by dsyme** [comment]
@KevinRansom I applied the suggested edits

**Pull request merged**